pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 
 ## Debugging Device Status
 
 If you find your device to be displaying a status which is unclear even with the above notes, visit the [device debugging masterclass][debugging-masterclass] for more information.
 
+# Update Status
+
+The `Update status` and `Update status duration` fields on devices represent the state of the current application update to target state. 
+
+__Note:__ Data for these states will only get populated for devices with supervisor 16.7.0 or later.
+
+## Update statuses
+
+| Status                                       | Description                                                                                                                                                                      |
+|----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Done**                              | The update, restart or configuration change is complete and the device is in its target state.                                          |
+| **Downloading**                       | The supervisor is downloading one or more container images associated with the target release                                           |
+| **Downloaded**                        | All container images in the target release have been downloaded. **Note:** If the device holds an update lock, it will stop in this state until the lock is released (depending on the [update strategy](https://docs.balena.io/learn/deploy/release-strategy/update-strategies/#controlling-the-update-strategy)). |
+| **Applying Changes**                  | The supervisor is performing actions to install the new release, i.e. killing old containers and making volumes/network/container changes.    |
+| **Rejected**                          | The target release has required services that do not meet the [necessary requirements](https://docs.balena.io/learn/develop/container-contracts/#container-contracts) to run. |
+| **Aborted**                           | The supervisor has given up on trying to install a new release. There are currently no conditions under which the supervisor returns this state, but future work could allow the supervisor to abort an update, for instance, after a number of failed download tries.    |
+
+## Update Status Duration
+
+The `Update Status Duration` field is calculated based on the timestamp of the most recent update event. When querying this data from the API, you will receive a timestamp instead of a duration.
+
+It is important to note that when the application starts up after a reboot or an app restart, this behavior is indistinguishable from an app installation. Consequently, the `Update Status Duration` timestamp will be refreshed whenever a reboot or restart occurs.
```

Comment:
This is potentially confusing - does this mean most devices in a stable state will say Done with an update status that's essentially the device or service uptime? I would expect (with this wording) for it to show the time since the update completed. Unsure what to suggest as an alternative at the moment 🤔

Reply:
@chrisys yeah unfortunately that is the case. If you rebooted a device, its `update status duration` would be equal to the "uptime" (minus supervisor boot time). I spoke to felipe about this but there isn't a simple way for the supervisor currently to distinguish between starting a composition of containers because of an update or because of restart/reboot. We will need to do a fairly large restructure of the supervisor for that :/